### DISCUSSION TITLE
Default domain for custom aliases

### DISCUSSION THREAD

**Discussion by erouiwjd23** [simple-login/app#641] [Feature Requests]
I'm a very happy paid customer and love the service.

As the title suggests, it would be great to have the option to select a default domain for **custom** aliases just like the option that already exists for **random** aliases. I often create custom addresses (using the name of the site I'm on, etc.), and with my list of many domains, I find myself always having to open the dropdown and select one of the "in-house" domains.

Thanks! :)

**Comment by nguyenkims** [comment]
@ibemanthony Hi, under the "Select the default domain for aliases" setting, you can select the default domain. It will then be used as default domain for both custom and random aliases :).
